routes/web.php

```diff
@@ -50,9 +60,16 @@
     #LOGINED ADMIN ONLY
     Route::group(['middleware' => 'admin', 'prefix' => 'admin', 'as' => 'admin.'], function () {
         Route::get('/',[AdminController::class,'showUsers'])->name('showUsers');
-        Route::get('/events',[AdminController::class,'showEvents'])->name('showEvents');
-        Route::get('/events/create',[AdminController::class,'createEvent'])->name('createEvent');
-        Route::post('/events/store',[AdminController::class,'storeEvent'])->name('storeEvent');
+      
+        Route::group(['prefix' => 'events'],function(){
+            Route::get('/',[AdminController::class,'showEvents'])->name('showEvents');
+            Route::get('/create',[AdminController::class,'createEvent'])->name('createEvent');
+            Route::post('store',[AdminController::class,'storeEvent'])->name('storeEvent');
+            Route::get('edit/{event}',[AdminController::class,'editEvent'])->name('editEvent');
```

Comment:
make the `/{event}` go before the `edit`. it should be like `/{event}/edit`

Reply:
I fixed it.